kafka-plugins-client/src/test/java/io/cdap/plugin/kafka/AbstractKafkaBatchSourceTest.java

```diff
@@ -95,6 +95,7 @@ public abstract class AbstractKafkaBatchSourceTest extends HydratorTestBase {
 
   @Before
   public void setup() throws Exception {
+    clear();
```

Comment:
Is this required ? It is better if tests clear the artifacts in finally block.

Reply:
Yes, need to clear the artifacts before running the unit test, otherwise execution of the second unit test gives this exception: `ArtifactAlreadyExistsException`.

If I clear the artifacts in the finally block at the end in each unit test, then I get this exception: `java.lang.RuntimeException: io.cdap.cdap.common.ApplicationNotFoundException: 'applicationreference:default.testKafkaSource' was not found.`

Reply:
Does both tests work with this in a single attempt ?

Reply:
Yes.

-------------------------------------------------------------------------------
Test set: io.cdap.plugin.kafka.KafkaBatchSourceTest
-------------------------------------------------------------------------------
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 65.571 sec